Backend/dev/migrations/dynamic-offer-driver-app/0165-reuse-old-estimate-in-repetition.sql

```diff
@@ -0,0 +1,135 @@
+CREATE TABLE atlas_driver_offer_bpp.search_request_2 (
+  id character(36) PRIMARY KEY NOT NULL,
+  transaction_id character(36) NOT NULL,
+  provider_id character(36) NOT NULL REFERENCES atlas_driver_offer_bpp.merchant (id),
+  from_location_id character varying(36) REFERENCES atlas_driver_offer_bpp.search_request_location (id),
+  to_location_id character varying(36) REFERENCES atlas_driver_offer_bpp.search_request_location (id),
+  bap_id character varying(255) NOT NULL,
+  bap_uri character varying(255) NOT NULL,
+  estimated_distance int NOT NULL,
+  estimated_duration int NOT NULL,
+  auto_assign_enabled boolean NOT NULL,
+  customer_language character varying(36),
+  device character varying(255),
+  created_at timestamp with time zone  NOT NULL DEFAULT now()
+);
+ALTER TABLE atlas_driver_offer_bpp.search_request_2 OWNER TO atlas_driver_offer_bpp_user;
+
+WITH SearchRequests AS (
```

Comment:
Lets add two backfill queries, one is backfilling the data of last 6 hours and other backfilling the data before that 6 hours so that our db load doesn't exceed threshold while backfilling as for backward compatibility we need last 6 hours data only.

Reply:
And move the query of backfilling whole data before the 6 hours to after rollout migrations;